Sources/Amplitude/Amplitude.m

```diff
@@ -284,6 +280,82 @@ - (instancetype)initWithInstanceName:(NSString *)instanceName {
     return self;
 }
 
++ (NSString *)getDataStorageKey:(NSString *)key {
+    NSString *dataStorageKey = [NSString stringWithFormat:@"%s_%@_%@_%@", "amplitude", _bundleIdentifier, self.instanceName, key];
+    return dataStorageKey;
+}
+
+- (void)migrateToFileStorage {
+    NSString *fileStoragePath =  [AMPStorage getAppStorageAmpDir:self.instanceName];
+    BOOL isDir;
+    [[NSFileManager defaultManager] fileExistsAtPath:fileStoragePath isDirectory:&isDir];
+    if (isDir) return;
+
+    NSString *databaseDirectory = [AMPUtils platformDataDirectory];
+    NSString *databasePath = [databaseDirectory stringByAppendingPathComponent:@"com.amplitude.database"];
```

Comment:
We should standardize the string here, "com.amplitude.database". I believe other people like Segment put them in Constants file. This is just for cleaner code when someone updates it later.